docs/Using-SRIOV-and-Clear-Containers.md

```diff
@@ -0,0 +1,265 @@
+# Setup to use SR-IOV with Clear Containers
+
+Single Root I/O Virtualization (SR-IOV) enables splitting a physical device into
+virtual functions (VFs). This enables direct passthrough to virtual machines or
```

Comment:
What do you gain here. I think the previous keeps the sentence short and clearer.

Reply:
I think we've already discussed this sentence: with a comma, it feels a bit "unwieldy" to me as it's (a) quite long, and (b) introduces two terms that take a bit of thinking about.

I do feel that keeping it as currently written is clearer as the period gives you time to consider before moving on to the next one. But it's just my opinion so will defer to the experts.

Reply:
Agreed that the comma makes the sentence quite long. Optional change to get rid the demonstrative pronoun, "This":  ...virtual functions (VSs). Splitting a physical device enables direct passthrough to virtual machines or containers. Although, this may change the entire meaning.

Reply:
Getting rid of demonstrative pronouns is always a win in my book. +1